tests/tangurnis_tala_gmg.prm

```diff
@@ -0,0 +1,331 @@
+# TanGurnis benchmark TALA
+
```

Comment:
instead of copying all this, can you replace it by the following:
```
# TanGurnis benchmark TALA using geometric multigrid preconditioner

include $ASPECT_SOURCE_DIR/benchmarks/tangurnis/tala/tan.prm 

Make all the GMG modifications here
```